packages/orbit-components/src/Tooltip/Tooltip.stories.js

```diff
@@ -93,6 +93,21 @@ storiesOf("Tooltip", module)
         "You can try all possible configurations of this component. However, check Orbit.Kiwi for more detailed design guidelines.",
     },
   )
+  .add(
+    "Full-width",
```

Comment:
```suggestion
    "FullWidth",
```

Reply:
I want to write this in natural language. I thought that "full-width" was the correct way to write it, but "full width" is correct, so I fixed it.